### ISSUE TITLE
Metadata layer, part 2

### ISSUE THREAD

**Issue by dcwhite** [SCIInstitute/SCIRun#972]
Follow-up of #896. 
- [ ] Metadata ports
- [ ] Metadata connections
- [ ] Metadata conversion/display module (with string outputs, for example)
- [ ] More metadata objects
- [ ] Save/load metadata to session file

**Comment by jessdtate** [comment]
I really like this idea too.  I think it can be a part of many projects, but I'm not sure it would be a priority in many of them.

**Comment by dcwhite** [comment]
@jessdtate Any UQ interest?

**Comment by jessdtate** [comment]
Yeah, I think this could be used to carry something about data distributions, or something similar.

**Comment by dcwhite** [comment]
**Triage 2026-06:** Metadata layer part 2 — metadata ports, connections, conversion/display modules. The metadata layer (#896) was a concept for piping non-field metadata through networks. No implementation has started. This is a large architectural addition. Still relevant for carrying provenance data and field annotations. High effort; depends on agreement on the metadata type system.
